### PR TITLE
Create guest developer user & agency

### LINKED ISSUE #197: Create guest developer user & agency for Hacktoberfest
This functionality is to support users creating their own agency-admin user in a shared instance of the O-FISH Realm app.

There is a matching web issue for this feature: https://github.com/WildAid/o-fish-realm/issues/136

- [ ] There should be a new config flag to enable/disable this new page (unauthorised error if the flag isn't set).
- [ ] There needs to be a new page `#/users/new_dev_user` that's based on `#/users/new_user`.

The page should look something like this (this is a "backdoor" for developers and so the design doesn't need to be precise):

image

All of the fields are free text.

When the button is pressed the following should be done:

- [ ] Log into Realm using anonymous authentication
- [ ] Call the Realm function `regDevUserAgency` passing in first name, last name, email, agency name and agency url as arguments(in that order
- [ ] If the function returns an error then display that error (errors could include "Agency already exists" or "User already exists", ...)
- [ ] Create the Realm username/password user (but **do not** add a document to the `User` collection as that's already been done by the `regDevUserAgency` function
- [ ] If the function returns successfully then automatically log in as that user and redirect to `/#/home` (or redirect to the login page if that's a lot simpler

### PR DESCRIPTION
**Pull request by brittanylau**
Fixes #197 

### Changes made
- [x] implemented `/users/new_dev_user` page
  - [x] log into Realm using anonymous authentication
  - [x] call the Realm function `regNewAgency`
  - [x] display errors from `regNewAgency`
  - [x] create the Realm username/password user (without adding document to `User` collection)
  - [x] if successful, log in as that user and redirect to `/home`
- [x] new config flag `sandbox` to enable/disable the `/users/new_dev_user` page
- [x] unauthorized error if the flag isn't set

### Screenshot
Screen Shot 2020-09-27 at 12 08 26 AM

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const config = {
     "citations-and-warnings":{
       chartId: "chart-id"
     }
-  }
+  },
+  sandbox: false,
```
Comment (brittanylau) [inline src/config.js:27]: Took a little creative liberty with naming this - open to other names!
The flag should be set as true when enabling the feature.

**Comment by o-fish-wildaid** [comment]
[Review PRs](https://trello.com/c/QKVtiwN0/372-review-prs)

**Review by Sheeri — APPROVED** [review]
Tested locally, going directly to 
http://localhost:3000/#/users/new_dev_user

I was able to fill out the form and create an agency. We don't really need to ask for website, but I guess there's no harm in it being there.

**Review by am-MongoDB — APPROVED** [review]

**Comment by am-MongoDB** [comment]
Confirmed that the agency and user are correctly setup in the backend.

Also confirmed that the user can log into the mobile app and create new reports which are then sync'd to Atlas.

**Pull request merged**